data/cloud_services.yaml

```diff
@@ -20,12 +20,14 @@ sources:
     paths:
     - '%%users.appdata%%\Dropbox\*.db*'
     - '%%users.localappdata%%\Dropbox\*.db*'
+    - '%%users.localappdata%%\Dropbox\instance1\sync_history.db'
```

Comment:
instance1 hints that there might be more instances, do you know if this can be the case?

Reply:
An instance2 folder is created when you log out and log in with a different account.   

I've updated the definition to glob after ```instance```.